### ISSUE TITLE
Absolutely not an issue. Using it on Android browsers: possible?

### ISSUE THREAD

**Issue by Guybrush3pw00d** [DevLARLEY/WidevineProxy2#6]
My best regards. As written in the title, nothing wrong so no complaints at all. It's clear from the outset that the extension is intended to be used on desktop browsers and it works like a charm there.
I'm just wondering if there is a way to use it on Android, even with a manual approach thinkering with the filesystem (I'm rooted). I've tested it both on Kiwi Browser (Chromium) and Fennec/Iceraven (Fenix): on Kiwi the installation just fails (I don't know why given that I've no issues with installing other desktop extensions) while both on Fennec and Iceraven the outcome is the same (not a surprise given that they have the same base): the installation goes fine but the UI doesn't work (a surprise neither) in the sense that it shows but when I click the button to load the wvd it just closes the UI and nothing happens.
On Windows I use WidevineProxy2 on Edge and it works great but I've also thought about installing it under Firefox too, check its extensions storage space and put everything inside the extension storage space on Android but here a major issue arise: I expect what I've called the extensions storage space to be inside the profile folder (which I know where to look for under Windows) while I haven't a bare idea about where to look on Android, maybe /data/data/etc and I can given that, as said, I'm rooted. I need to check but I don't know if such an approach would ever work out.
I hope my input could be interesting but I apologize in advance shouldn't it be te case. Have a nice day! :)

**Comment by fawyrad** [comment]
Read on the closed issue. You will find the solution to fix the error not be able to import the extension to Kiwi Browser. Cheers.

**Comment by Guybrush3pw00d** [comment]
Thank you very much for sharing your solution. My bad I haven't checked  the closed issues pane before, my sincere apologies.
I would apply the workaround ASAP but I'm already sure everything would be fine.
Any clues about the described 'issue' with Fenix browsers? Do you think that my idea of getting a wvd already installed in a desktop instance to avoid using the not working button could help?
Thanks again and have a nice day.

**Comment by kereminggat** [comment]
simple if you want to use it on android browser like kiwi, edit manifest.json delete line 25.

**Comment by Guybrush3pw00d** [comment]
I can confirm that I've got it working like a charm on Kiwi following the precious advices from @fawyrad and @kereminggat. I'm instead struggling to get it working on Fenix browsers (which experience the issue with the UI described above). 
I've tried to take a look at the code to see if I could figure out how to put the wvd in place manually (in the right place, that's obvious but I don't have the necessary insight: I've pinpointed the relevant piece of code is picker/wvd/filePicker.js but I couldn't go any further than that and I've given up.
That's a shame given that it could be that someone (not me as I like Kiwi a lot) doesn't like to install Kiwi (or any other Chromium browser) while he likes a Fenix one (many of them out there) but that's how it is. Thanks to all of you for your precious help! :)
